### PR TITLE
Add more flare data

### PR DESCRIPTION
**Pull request by directionless**
Add some helpers to pull files and commands. Update some flares to fetch more data. I think the diff should be fairly self explanatory

### CODE REVIEW

**Review by RebeccaMahany — DISMISSED** [review]
LGTM, just some small comments

File: pkg/debug/checkups/util.go [hunk 1]
```diff
@@ -28,3 +35,121 @@ func recursiveDirectoryContents(extraFH io.Writer, basedir string) (int, error)
...
+}
+
+// addFileToZip takes a file path, and a zip writer, and adds the file and some metadata.
+func addFileToZip(z *zip.Writer, location string) error {
+	metaout, err := z.Create(filepath.Join(".", location+".flaremeta"))
+	if err != nil {
+		return fmt.Errorf("creating %s in zip: %w", location+".flaremeta", err)
+	}
+
+	// Not totally clear if we should use Lstat or Stat here.
```
[the remark this answers is not in the document]
Comment (directionless) [reply pkg/debug/checkups/util.go:54]: I can't decide if it's interesting to know if it's a link. I opted for stat for the reason you outlined

File: pkg/debug/checkups/util.go [hunk 2]
```diff
@@ -28,3 +35,121 @@ func recursiveDirectoryContents(extraFH io.Writer, basedir string) (int, error)
...
+			for _, prefix := range ignoredEnvPrefixes {
+				if strings.HasPrefix(e, prefix) {
+					skip = true
+					break
+				}
+			}
+			if skip {
+				continue
+			}
+			fmt.Fprintf(extraWriter, "export %s\n", e)
```
[the remark this answers is not in the document]
Comment (directionless) [reply pkg/debug/checkups/util.go:129]: For now, I think I want to leave it as is. I'm not sure what correct will be, and this felt easy to cut and paste.

We might decide to drop the markdown, and move to something like `command.out` and `metadata.json` but 🤷

File: pkg/debug/checkups/util.go [hunk 3]
```diff
@@ -28,3 +35,121 @@ func recursiveDirectoryContents(extraFH io.Writer, basedir string) (int, error)
...
+type fileInfo struct {
+	Name    string    // base name of the file
+	Size    int64     // length in bytes for regular files; system-dependent for others
+	Mode    string    // file mode bits
+	ModTime time.Time // modification time
+	IsDir   bool      // abbreviation for Mode().IsDir()
+}
+
+// addFileToZip takes a file path, and a zip writer, and adds the file and some metadata.
+func addFileToZip(z *zip.Writer, location string) error {
```
[the remark this answers is not in the document]
Comment (directionless) [reply pkg/debug/checkups/util.go:48]: 🙌 

I had this idea that we should go a step further, and move the files from being stored in the individual flares, to being stored centrally. But I didn't want that much refactoring just yet.

**Review by RebeccaMahany — APPROVED** [review]

**Pull request merged**